ambry-api/src/test/java/com.github.ambry/rest/MockRestResponseChannel.java

```diff
@@ -245,6 +249,11 @@ private void setHeader(String headerName, Object headerValue, Event eventToFire)
           if (!responseMetadata.has(RESPONSE_HEADERS_KEY)) {
             responseMetadata.put(RESPONSE_HEADERS_KEY, new JSONObject());
           }
+          if (headerValue instanceof Date) {
```

Comment:
what is the purpose of doing this. Already the request will have header value set as Date if I am not wrong.